django_cloud_deploy/workflow/_deploygae.py

```diff
@@ -49,9 +51,10 @@ def deploy_gae_app(self,
         args = ['app', 'deploy', app_yaml_path, project]
         process = pexpect.spawn('gcloud', args)
         try:
-            index = process.expect(
-                ['\[{}\]\s*{}'.format(i, region) for i in range(1, 10)])
-            process.sendline(str(index))
+            if is_new:
```

Comment:
In a follow-up PR, could you just run gcloud with the '-q' flag to remove all this prompting stuff?

Reply:
We tried, it doesn't work.